test/e2e/storage/volumes.go

```diff
@@ -138,10 +138,13 @@ var _ = framework.KubeDescribe("Volumes [Volume]", func() {
 	// Gluster
 	////////////////////////////////////////////////////////////////////////
 
-	framework.KubeDescribe("GlusterFS [Feature:Volumes]", func() {
+	framework.KubeDescribe("GlusterFS", func() {
 		It("should be mountable", func() {
-			//TODO (copejon) GFS is not supported on debian image.
-			framework.SkipUnlessNodeOSDistroIs("gci")
```

Comment:
When (48286)[https://github.com/kubernetes/kubernetes/pull/48286] is merged then can this line be left as is?